### ISSUE TITLE
validate CC with checkmark

### ISSUE THREAD

**Issue by dmikester1** [wangjohn/creditly#8]
I love this library!  Any chance you could add an option to display a green checkmark in the card field after the user has typed a valid number?  The previous code that I was using did this.  It would fire on each keypress and check the Luhn Algorithm I believe.

**Comment by wangjohn** [comment]
Good idea! This will definitely be going on the list of things to implement in the coming weeks.
